### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by benvand**
[flake8-report.txt](https://github.com/alphagov/digitalmarketplace-aws/files/1337227/flake8-report.txt)

### CODE REVIEW

File: scripts/clean-db-dump.sql [hunk 1]
```diff
@@ -144,27 +144,22 @@ SELECT
         '"respondToEmailAddress": "example-email@gov.uk"' ||
       '}'
     )::JSON,
-    eligible_brief_supplier_pairings.brief_id,
+        eligible_brief_supplier_pairings.supplier_id,
```
Comment (samuelhwilliams) [inline scripts/clean-db-dump.sql]: This doesn't seem like a flake8 change? It looks over-indented, it changes a key, and it duplicates the key below?

Comment (samuelhwilliams) [reply scripts/clean-db-dump.sql]: Are any of the changes in this file intentional?

Comment (benvand) [reply scripts/clean-db-dump.sql]: Damn, they snuck in there, good spot.

Comment (benvand) [reply scripts/clean-db-dump.sql]: Done 👍

**Pull request merged**